.github/workflows/examples-components.yml

```diff
@@ -429,15 +402,16 @@ jobs:
           echo "bin-name=wash-build-${{ matrix.wash-version }}-${{ matrix.project.lang }}-component-${{ matrix.project.folder }}/${{ matrix.project.wasm-bin }}" >> $GITHUB_OUTPUT;
           echo "ref-prefix=$REF_PREFIX" >> $GITHUB_OUTPUT;
           echo "wasmcloud-toml-path=examples/${{ matrix.project.lang }}/components/${{ matrix.project.folder }}/wasmcloud.toml" >> $GITHUB_OUTPUT;
-      # Download all artifacts (wash binary and example component binaries) to work dir
+      # Download all artifacts (example component binaries) to work dir
       - uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16
         with:
           path: artifacts
-      - name: install wash binary to PATH # Some build tools expect wash to be on the PATH
-        shell: bash
-        run: |
-          chmod +x ./artifacts/wash-${{ matrix.wash-version }}/wash;
-          echo "$(realpath ./artifacts/wash-${{ matrix.wash-version }})" >> "$GITHUB_PATH";
+      - name: install wash (current)
+        uses: taiki-e/cache-cargo-install-action@caa6f48d18d42462f9c30df89e2b4f71a42b7c2c # v2.0.1
+        with:
+          tool: wash-cli
+          git: https://github.com/${{ github.repository }}
```

Comment:
You could use [`${{ github.repositoryUrl }}`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#example-contents-of-the-github-context) for this